### PR TITLE
mgr/dashboard: Grafana dashboard updates and additions

### PR DESCRIPTION
**Pull request by pcuzner**
This PR tidies up some anomalies (editable flags) in some of the dashboards, and add the overview dashboards for the Hosts and OSD sections of the ceph dashboard UI

Signed-off-by: Paul Cuzner <pcuzner@redhat.com>

### CODE REVIEW

**Comment by b-ranto** [comment]
@pcuzner Can you update these in a similar fashion that the `ceph-cluster.json` uses datasource? The `ceph-cluster.json` defines `$dashboard` variable in templating and then uses the variable as a datasource. The variable queries for all the available prometheus datasources. This makes the datasource configurable if there is more than a single prometheus datasource.

File: monitoring/grafana/README.md [hunk 1]
```diff
@@ -8,5 +8,7 @@ plugin](http://docs.ceph.com/docs/master/mgr/prometheus/) and the
 
 ### Other requirements
 
-- Mimic or newer
+- Luminous 12.2.5 or newer
```
[the remark this answers is not in the document]
Comment (pcuzner) [reply monitoring/grafana/README.md]: Thanks Jan. I'll reply in the tracker.

**Comment by pcuzner** [comment]
@b-ranto Is there a label name that has been defined for this? Happy to make the change, but I'd prefer to do it just the once for each dashboard

File: monitoring/grafana/README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (b-ranto) [reply monitoring/grafana/README.md]: As long as data exporting goes, the prometheus module in luminous should be on par with what we have in master. We might need a later version (12.2.8 probably) for it to be true though.

**Comment by b-ranto** [comment]
@pcuzner You only need to do it once for dashboard. You just need to define `$datasource` templating variable like so

https://github.com/ceph/ceph/blob/master/monitoring/grafana/dashboards/ceph-cluster.json#L1108

and then you can use `datasource: $datasource` everywhere else:

https://github.com/ceph/ceph/blob/master/monitoring/grafana/dashboards/ceph-cluster.json#L1023

**Comment by pcuzner** [comment]
@b-ranto yes I know how templating works :) My point was what's the label name for the datasource variable that will appear on all the dashboards...do we call it "Prometheus", or simply "Datasource"

**Comment by b-ranto** [comment]
@pcuzner Ah, yeah, with the definition that is currently in `ceph-cluster.json` it shows up as `datasource` but I would be fine either way (as long as it is consistent). I suppose it is falling back to the name of the template variable when you don't give it any label name.

**Comment by a2batic** [comment]
@jan--f @b-ranto , please provide your review for the pull request.

**Comment by jan--f** [comment]
Ok...I guess were not going for or discussing a uniform style?

**Pull request merged**